app/presenters/single_content_item_presenter.rb

```diff
@@ -68,15 +68,10 @@ def feedex_context
 
   def feedback_explorer_href
     host = Plek.new.external_url_for('support')
-    template = Addressable::Template.new(
-      "#{host}/anonymous_feedback{?from,to,paths}"
-    )
-    query = {
-      paths: base_path,
-      from: @date_range.from,
-      to: @date_range.to
-    }
-    template.expand(query).to_s
+    from = @date_range.from
+    to = @date_range.to
+    path = CGI.escape(base_path)
+    "#{host}/anonymous_feedback?from=#{from}&to=#{to}&paths=#{path}"
```

Comment:
Hot tip: `{
      paths: base_path,
      from: @date_range.from,
      to: @date_range.to
    }.to_query` will give you a query string, with the automatically arguments escaped.

https://apidock.com/rails/ActiveSupport/CoreExtensions/Hash/to_query